08-walkthrough-3.Rmd

```diff
@@ -27,91 +31,79 @@ We take up the following four questions:
+The setting of this study was a public provider of individual online courses in a Midwestern state. In particular, the context was two semesters (Fall and Spring) of offerings of five online science courses (Anatomy & Physiology, Forensic Science, Oceanography, Physics, and Biology), with a total of 36 classes. 
 
 Specific information in the dataset included:
 
 - a pre-course survey students completed about their self-reported motivation in science — in particular, their perceived competence, utility value, and interest
-- the time students spent on the course (obtained from the LMS, Blackboard) and their final course grades as well as their involvement in discussion forums
-- for discussion board responses, we used the Linguistic Inquiry and Word Count (LIWC; Pennebaker, Boyd, Jordan, & Blackburn, 2015) to calculate the number of posts per student and variables for the mean levels of students' cognitive processing, positive affect, negative affect, and social-related discourse
----
-title: 'Education Dataset Analysis Pipeline: Walk Through #3'
-output: html_document
----
+- the time students spent on the course (obtained from the learning management system (LMS), Blackboard
+- students' final course grades 
+- students' involvement in discussion forums
 
-```{r analysis-preferences}
-# Seed for random number generation
-set.seed(42)
-knitr::opts_chunk$set(cache.extra = knitr::rand_seed, eval = TRUE, echo = FALSE, results = 'hide',
-                      message = FALSE, warning = FALSE)
-```
-
-# Background
-
-One area of interest is the delivery of online instruction, which is becoming more prevalent: in 2007, over 3.9 million U.S. students were enrolled one or more online courses (Allen & Seaman, 2008). 
-
-In this walkthrough, we examine the educational experiences of students in online science courses at a virtual middle school in order to characterize their motivation to achieve and their tangible engagement with the course in terms of behavioral trace measures. To do so, we use a robust data set, which includes self-reported motivation as well as behavioral trace data collected from a learning management system (LMS) to identify predictors of final course grade. Our work examines the idea of educational success in terms of student interactions with an online science course.
+For discussion board responses, we used the Linguistic Inquiry and Word Count (LIWC; Pennebaker, Boyd, Jordan, & Blackburn, 2015) to calculate the number of posts per student and variables for the mean levels of students' cognitive processing, positive emotions, negative emotions, and social-related discourse.
+<<<<Josh, is this preceding paragraph jargony>>>>
```

Comment:
I don't think this is necessarily jargony, but the tone is academic. one option for rephrasing it would be to say "we are interested in calculating.... the number of posts per student...." and then end by introducing the LIWC, followed by a brief description of the LIWC.

Reply:
Great idea - thanks.